pkg/util/mount/mount_windows.go

```diff
@@ -204,3 +242,28 @@ func getDriveLetterByDiskNumber(diskNum string, exec Exec) (string, error) {
 	}
 	return string(output)[:1], nil
 }
+
+// getAllParentLinks walks all symbolic links and return all the parent targets recursively
+func getAllParentLinks(path string) ([]string, error) {
+	links := make([]string, 2)
+	for {
+		fi, err := os.Lstat(path)
+		if err != nil {
+			return links, fmt.Errorf("Lstat: %v", err)
+		}
+		if fi.Mode()&os.ModeSymlink == 0 {
+			break
+		}
+
+		path, err = os.Readlink(path)
+		if err != nil {
+			return links, fmt.Errorf("Readlink error: %v", err)
+		}
+		links = append(links, path)
+		if len(links) > 10 {
```

Comment:
where does this limit come from?

Reply:
actually the number should be 255, I will change to 255. see:
https://github.com/golang/go/blob/master/src/path/filepath/symlink.go#L37